Makefile

```diff
@@ -196,9 +196,12 @@ verify:
 	@echo "verify: Started"
 	@echo "verify: Completed"
 
-test:
-	@echo "test: Started"
-	@echo "test: Completed"
+SKIP_TESTS ?=
+test: export LD_LIBRARY_PATH=$(VDDK_LIBS)
+test: build-dirs
+ifneq ($(SKIP_TESTS), 1)
+	hack/test.sh $(WHAT)
```

Comment:
1. Use much more meaningful variable here than `WHAT`.
2. Run test in a docker shell rather than in local, so that developers can do `make test` in any OS platform.